cumulusci/cumulusci.yml

```diff
@@ -219,7 +219,6 @@ tasks:
         description: Merges the latest commit on a parent feature branch into all child feature branches
         class_path: cumulusci.tasks.github.MergeBranch
         options:
-            children_only: True
             source_branch: $project_config.repo_branch
         group: GitHub
```

Comment:
Should we rename this task to `github_automerge_feature` like I suggested in the first automerge RFC? It's no longer just handling parent-to-child.

Reply:
@davisagli - I'm not sure I agree with the name change. This task's primary purpose is to facilitate the auto-merging from "Parent to Child" branches. We've added an option which allows for updating of future releases, but this is feels more secondary to the parent-child automerges. I would prupose: `github_automerge_main_to_feature` and `github_automerge_parent_to_child` for the names of the two flows. Let me know what you think.

Reply:
I thought it might be a bit confusing to call it parent_to_child when it does more than that. But I'm okay with it.

If we're going in that direction, I'd suggest github_main_to_feature and github_parent_to_children to minimize changes from the status quo.